Multiple doc updates for CR docs (#9556)
harness/developer-hub: docs/code-repository/config-repos/webhooks.md

> ## Create a webhook

Reviewer: @johannesHarness how about https://app.harness.io/ng/account/ACCOUNT_ID/module/code/orgs/default/projects/PROJECT_ID/settings/webhooks? I can view this in UI.

> ## Create a webhook

Reviewer 2: @dewan-ahmed ohh I see - those aren't repo webhooks. as a matter of fact, I don't think they are actual webhooks - you can configure them to act/listen on webhooks as far as I understand (e.g. they don't actually trigger anything as you'd expect from a webhook)

I can start a thread to see where we are with the UI on that, but I'm pretty sure it doesn't exist

> In Harness Code, you can create webhooks for a specific repository, project, org, or the entire account. 

Reviewer 2: this sounds a bit like the webhook is created for a project. e.g. it is called on project events.
Maybe we should say sth like
```suggestion
In Harness Code, you can create webhooks at the repository, project, organization, or account level. Webhooks created at a higher level (such as account or organization) are triggered for events from all repositories within that level, as well as those in any contained projects or organizations.
```

> In Harness Code, you can create webhooks for a specific repository, project, org, or the entire account. 

Reviewer 2: ```suggestion
In Harness Code, you can create webhooks at the repository, project, organization, or account level.
```

that way it doesn't sound like you create a webhook for a project. it's still a repo webhook, but on a project level

> In Harness Code, you can create webhooks at the repository, project, organization, or account level.
> 
> While webhooks created at the repository level only apply to that specific repository, webhooks created at a higher level (such as account or organization) are triggered for events from all repositories within that level, as well as those in any directly contained projects or organizations.
> 

Reviewer: ```suggestion
```

> Webhooks set on a repository only apply to that specific repository but you can also create webhooks at the project, organization or account level to send a change payload when any repository in that container is updated. E.g. A webhook set on a project will send a payload for every repository change in that project - even newly created repositories. 

Reviewer 2: ```suggestion
While webhooks created at the repository level only apply to that specific repository, webhooks created at a higher level (such as account or organization) are triggered for events from all repositories within that level, as well as those in any directly contained projects or organizations.

```

You can create ad hoc webhooks to integrate your repository with other services, such as notification services or third party CI/CD providers.

> 1. When viewing a repository or a list of repositories in a project, org or the account in Harness Code, go to **Webhooks**, and select **New Webhook**.

Reviewer: ```suggestion
1. When viewing a repository in Harness Code, go to **Webhooks**, and select **New Webhook**.
```
